files/tasks/install-ogr-deps.yaml

```diff
@@ -0,0 +1,23 @@
+# Install dependencies from setup.cfg as rpm packages for ogr
+- name: Download script setupcfg2rpm.py
+  get_url:
+    url: https://raw.githubusercontent.com/packit-service/deployment/master/scripts/setupcfg2rpm.py
+    dest: ./setupcfg2rpm.py
+    mode: "0744"
+- name: Download ogr setup.cfg
+  get_url:
+    # ogr has only master branch
+    url: https://raw.githubusercontent.com/packit-service/ogr/master/setup.cfg
+    dest: ./ogr_setup.cfg
+- name: Install ogr dependencies provided by setupcfg2rpm
+  shell: dnf install $(./setupcfg2rpm.py ogr_setup.cfg) -y
+  args:
+    warn: no
+  become: true
+- name: Install pip deps
+  pip:
+    name:
+      - git+https://github.com/packit-service/sandcastle.git
+      - sentry-sdk
+      - flask-restx
```

Comment:
There are not ogr dependencies, are they? Then why they are here (in `install-ogr-deps.yaml`)?

Reply:
looks like I copied more than I wanted - will remove it

Reply:
fixed